src/app/components/header/header.component.tsx

```diff
@@ -1,9 +1,25 @@
+import { NavBar } from '../../app';
 import './header.component.scss';
 
 interface HeaderProps {
     appName: string;
 }
 
-export function Header({appName}: HeaderProps) {
-    return <div className='abc'>hello header</div>
+function Header({ appName }: HeaderProps) {
```

Comment:
no need to create separate component for wrapping an appName; moreover it has name of element from html5 specification

Reply:
Header removed, appName prop moved into MainHeader